### PR TITLE
Page transporter and web element actions

### PR DESCRIPTION
**Pull request by edsonarios**

### CODE REVIEW

**Review by SaulCM — APPROVED** [review]
Good job. Gotta fix checkstyle errors.

File: src/main/java/core/config/EnvValues.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ */
+public class EnvValues {
+    private ReadFile readFile;
+    private String path = "environmentVariables.properties";
+    private String path2 = "config.properties";
+
+    public EnvValues() {
+        this.readFile = new ReadFile(path);
+    }
+
```
Comment (Paola-Xaq) [inline src/main/java/core/config/EnvValues.java:15]: JavaDocs is missing

File: src/main/java/core/config/ConfigValues.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package core.config;
+
+import core.utils.ReadFile;
+
+/**
+ * Read config values.
+ */
+public class ConfigValues {
```
Comment (Paola-Xaq) [inline src/main/java/core/config/ConfigValues.java:8]: JavaDoc is missing in the methods, review your Checkstyle

File: src/main/java/salesforce/ui/PageTransporter.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+public class PageTransporter {
+    private WebDriver driver;
+    private ConfigValues configValues;
+
+    public PageTransporter(WebDriver driver) {
+        this.driver = driver;
+        configValues = new ConfigValues();
+    }
+
+    private void goToUrl(final String url) {
```
Comment (Paola-Xaq) [inline src/main/java/salesforce/ui/PageTransporter.java:20]: Still missing JavaDocs

File: src/main/java/salesforce/ui/pages/quicktext/NewQuickText.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+ */
+public class NewQuickText extends BasePage {
+
+    @FindBy(css = "input[aria-required=\"true\"]")
+    private WebElement quickTextNameFld;
+
+    @FindBy(id = "quickTextMessageInputTextArea")
+    private WebElement quickTextMessageFld;
+
+    @FindBy(css = ".slds-button:nth-child(4) > .label")
```
Comment (Paola-Xaq) [inline src/main/java/salesforce/ui/pages/quicktext/NewQuickText.java:20]: if its possible try to not use indexing with child

**Pull request merged**